### PR TITLE
add history command

### PR DESCRIPTION
**Pull request by ningmingxiao**
Signed-off-by: ningmingxiao <ning.mingxiao@zte.com.cn>

### CODE REVIEW

**Comment by ningmingxiao** [comment]
need help,I can't get image id，use snapshot instead.

File: cmd/nerdctl/container.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ func newContainerCommand() *cobra.Command {
 		newStopCommand(),
 		newStartCommand(),
 		newRestartCommand(),
+		newHistoryCommand(),
```
Comment (AkihiroSuda) [inline cmd/nerdctl/container.go:45]: This has to be under `nerdctl image`, not `nerdctl container`.

Also please update README.md

**Comment by AkihiroSuda** [comment]
> I can't get image id，use snapshot instead.

SGTM

File: cmd/nerdctl/container.go (hunk 1 above)
Comment (ningmingxiao) [reply cmd/nerdctl/container.go]: done,thank you! Happy spring  Festival

Comment (AkihiroSuda) [reply cmd/nerdctl/container.go]: Xinnian kuaile ㊗️ , could you rebase with the master to pass the CI?

Comment (AkihiroSuda) [reply cmd/nerdctl/container.go]: Ping @ningmingxiao 🙏

**Review by AkihiroSuda — APPROVED** [review]
Thanks

Merging, but ~please consider changing the variable name~ (opened a follow-up: https://github.com/containerd/nerdctl/pull/813)

**Pull request merged**